ENH: Add global nullable option (#50748)
pandas-dev/pandas: doc/source/whatsnew/v2.0.0.rst

* :func:`read_feather`
* :func:`to_numeric`

To simplify the global opt-in a new option ``nullable_dtypes`` was added that allows to set
the keyword argument globally to ``True`` if not specified directly. The option can be enabled
through:

.. ipython:: python

    pd.options.mode.nullable_dtypes = True

> The option will only work in context with the keyword ``use_nullable_dtypes``.

Reviewer: ```suggestion
The option will only work for functions with the keyword ``use_nullable_dtypes``.
```
